pkg/install/install.go

```diff
@@ -129,6 +137,7 @@ func (i *Installer) Install(ctx context.Context, installConfig *installconfig.In
 			i.createPrivateEndpoint,
 			i.updateAPIIP,
 			i.createCertificates,
+			i.initializeKubernetesClients, // may want to move this above later
```

Comment:
remove the comment.  I bet you need this line above `i.removeBootstrap,` as well?